hack/jenkins/e2e-runner.sh

```diff
@@ -49,6 +54,7 @@ if [[ "${E2E_UP,,}" == "true" || "${JENKINS_FORCE_GET_TARS:-}" =~ ^[yY]$ ]]; the
             exit 1
         fi
 
+        # This is for release jobs on GKE, once we branch the release
```

Comment:
this comment is still a little ambiguous, I think; this is for testing official releases on GKE, right? not the latest and greatest from a `release-X` branch?